app/controllers/volunteer_ops_controller.rb

```diff
@@ -57,24 +69,43 @@ def volunteer_op_params
 
   private
 
-  def build_map_markers(organisations)
-    ::MapMarkerJson.build(organisations) do |org, marker|
+  def build_map_markers(organisations,  type = :harrow, include_extra_organisation_data = true)
+    ::MapMarkerJson.build(organisations, include_extra_organisation_data) do |org, marker|
       marker.lat org.latitude
       marker.lng org.longitude
-      marker.infowindow render_to_string( partial: 'popup', locals: {org: org})
+      marker.infowindow render_to_string( partial: "popup_#{type}", locals: {org: org})
       marker.json(
         custom_marker: render_to_string(
-          partial: 'shared/custom_marker',
-          locals: { attrs: [ActionController::Base.helpers.asset_path("volunteer_icon.png"),
-                    'data-id' => org.id,
-                    class: 'vol_op', title: "Click here to see volunteer opportunities at #{org.name}"]}
+          partial: "shared/custom_marker",
+          locals: { attrs: [ActionController::Base.helpers.asset_path("volunteer_icon_#{ type }.png"),
+                    "data-id" => org.id,
+                    class: "vol_op", title: "Click here to see volunteer opportunities at #{ org.name }"]}
```

Comment:
Line is too long. [106/80]<br>Space inside } missing.<br>Space inside string interpolation detected.